### PR TITLE
mgr/send_report: Add a debug log for send_report function

### PR DESCRIPTION
**Pull request by YiteGu**
This log for daemon's type, name and metric when report health
check to mon. It help we that check mgr send message and mon
accept message whether consistent.

Signed-off-by: Yite Gu <ess_gyt@qq.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by YiteGu** [comment]
For example:
```
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report osd.0 PENDING_CREATING_PGS(0|(0,0))
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report osd.1 SLOW_OPS(0|(0,0))
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report osd.1 PENDING_CREATING_PGS(0|(0,0))
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report mon.a SLOW_OPS(0|(0,0))
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report mon.b SLOW_OPS(0|(0,0))
2022-08-25 14:48:02.160 7f568c39e700  4 mgr.server send_report mon.c SLOW_OPS(0|(0,0))
```

**Comment by YiteGu** [comment]
@rzarzynski have a code review :)

File: src/mgr/DaemonServer.cc [hunk 1]
```diff
@@ -2627,6 +2627,7 @@ void DaemonServer::send_report()
           tie(acc, std::ignore) = accumulated.emplace(metric.get_type(),
               std::move(collector));
         }
+        dout(4) << key << " " << metric << dendl;
```
Comment (rzarzynski) [inline src/mgr/DaemonServer.cc:2630]: Is there a reason for `4`. Similar debugs around are on `10` (info). Also, might we worth to print `__func__`.

**Comment by YiteGu** [comment]
@rzarzynski  this issue(https://tracker.ceph.com/issues/62701) seem relevant this PR

**Pull request closed without merging**